ui/perfherder/alerts/AlertTableRow.jsx

```diff
@@ -33,6 +32,7 @@ import {
 
 import AlertTablePlatform from './AlertTablePlatform';
 import AlertTableTagsOptions from './AlertTableTagsOptions';
+import AlertTableMagnitude from './AlertTableMagnitude';
```

Comment:
In the context of Perfherder `AlertTableMagnitude` sounds like this component is responsible for a whole table, but in reality it only handles a cell value from Alert's Table. I think it should have a different name, like `MagnitudeCellValue` / `MagnitudeComponent` or something similar.